AzureCore/Source/AzureKeys.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  AzureKeys.swift
+//  AzureCore
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import Foundation
+
+public class AzureKeys: Codable {
+    enum CodingKeys: String, CodingKey {
+        case accountName = "AzureAccountName"
+        case masterKey = "AzureAccountMasterKey"
+    }
+
+    public let accountName: String
+    public let masterKey: String
+
+    public var hasValidAccountName: Bool {
```

Comment:
obviously you'll need to update these property names to correspond to the updated cosmos-specific names